_board/microdev_micro_s2.md

```diff
@@ -1,39 +1,33 @@
 ---
 layout: download
 board_id: "microdev_micro_s2"
-title: "microDev microS2 Download"
+title: "microS2 Download"
 name: "microS2"
 manufacturer: "microDev"
-board_url: ""
+board_url: "https://github.com/microDev1/microS2/wiki"
 board_image: "microdev_micro_s2.jpg"
 date_added: 2020-08-26
 features:
   - Wi-Fi
   - STEMMA QT/QWIIC
 ---
 
-Introducing the microS2 - An ESP32-S2 based development board in the size of an esp32 module!
+![](https://raw.githubusercontent.com/microDev1/microS2/master/docs/microS2-img.png)
```

Comment:
I'm a little worried about using these images on the website. Ideally we'd have them in this repo and the URL would then be at circuitpython.org. Can you add them here?

Reply:
Got it... I removed the external links.